docs/userGuide/syntax/mathformulae.mbdf

```diff
@@ -0,0 +1,99 @@
+## Math Formulae
+Markbind supports typesetting TeX math equations. [KaTeX](https://katex.org) is used as a fast math renderer.
+
+<box type="info">
+
+Insert `inline` equations by enclosing them in round brackets `\( ... \)`.
+
+Insert `display` equations by enclosing them in square brackets `\[ ... \]`.
+
+Insert numbered `display` equations by enclosing the equation square brackets and the equation number in curly brackets `\[ ... \] (1)`.
+</box>
+
+<box type="important">
+
+When typing `display` equations, this does not work as well:
+
+```markdown
+abc
+\[2+2=4\]
+```
+
+or this
+```markdown
+> abc
+> \[2+2=4\]
+```
+
+If your equation requires special Nunjucks tags like {% raw %}`{{`{% endraw %} or {% raw %}`}}`{% endraw %},use a
+[raw-endraw block together with v-pre](https://markbind.org/userGuide/tipsAndTricks.html#using-raw-endraw-to-display-content):
+
+```markdown
+{% raw %}\(e^{{\frac{1}{3}} + 1}\){% endraw %} {v-pre}
```

Comment:
as mentioned `v-pre` _shouldn't_ be necessary, which is one of the benefits of doing it at compile time to. do double check though

Reply:
I'm not exactly sure why, but leaving out the {v-pre} causes a blank page to show:
![image](https://user-images.githubusercontent.com/38814428/92318420-aa0f7600-f03e-11ea-950b-ae0229032eaf.png)

Whereas leaving it in allows everything to load correctly, so I left it in for now.

Reply:
seems like katex also spits out MathML accessibility output retaining the original equation by default:

from view page source:
```
<semantics>
--
  | <mrow>
  | <msup>
  | <mi>e</mi>
  | <mrow>
  | <mfrac>
  | <mn>1</mn>
  | <mn>3</mn>
  | </mfrac>
  | <mo>+</mo>
  | <mn>1</mn>
  | </mrow>
  | </msup>
  | </mrow>
  | <annotation encoding="application/x-tex">e^{{\frac{1}{3}} + 1}</annotation>
  | </semantics>

```

let's auto add the `v-pre` to this in `componentParser`

```
<annotation encoding="application/x-tex">e^{{\frac{1}{3}} + 1}</annotation>
```